src/components/v5/common/ActionSidebar/partials/forms/SimplePaymentForm/hooks.ts

```diff
@@ -32,14 +32,18 @@ const validationSchema = yup
     decisionMethod: yup.string().defined(),
     payments: yup
       .array()
-      .min(1, 'it needs to have at least one element')
       .of(
         yup.object().shape({
-          recipent: yup.string(),
-          amount: yup.string(),
+          recipent: yup.string().required(),
+          amount: yup
+            .number()
+            .required()
+            .transform((value) => toFinite(value))
+            .moreThan(0),
```

Comment:
Is this validation correct? Look at line 17, I believe we use the same component for both fields